cypress/e2e/eda/Decision-Environments/decision-env-crud.cy.ts

```diff
@@ -69,8 +69,8 @@ describe('EDA decision environment- Create, Edit, Delete', () => {
       cy.clickModalButton('Delete decision environments');
       cy.wait('@deleteDE').then((deleteDE) => {
         expect(deleteDE?.response?.statusCode).to.eql(204);
-        cy.hasTitle(/^DecisionEnvironments$/);
       });
+      cy.hasTitle(/^Decision environments$/);
     });
```

Comment:
This change removes an assertion. if you want to make this change, let's update the custom command to include a specific assertion

Reply:
updated